.github/pyinstaller/pyinstaller.spec

```diff
@@ -17,6 +17,7 @@ a = Analysis(
         # when invoking pyinstaller from the project root,
         # this gets invoked from the directory of the spec file,
         # i.e. ./.github/pyinstaller
+        ("../../assets", "assets"),
```

Comment:
maybe we want to be more specific here (rename to `comdata`)
or create a sub directory to organize this if we envision more data to go into here (`assets/comdata`)?

Reply:
let's discuss before making all the trailing changes

Reply:
we could put signatures in `assets`, too. i think reorganization of the supporting data files can be done in a followup PR?

Reply:
I think this will be better. 
> create a sub directory to organize this if we envision more data to go into here (`assets/comdata`)?

In another PR #1737 , there are more file to be uploaded. We can move all supporting data files to subdirectories in `./assets`.